api/config/12-db-deployment.yaml

```diff
@@ -57,6 +57,16 @@ spec:
           volumeMounts:
             - name: db
               mountPath: /var/lib/postgresql/data
+          readinessProbe:
+            exec:
+              command: [psql, -w, -U, postgres, -d, $(POSTGRES_DB), -c, "SELECT 1"]
```

Comment:
if the variable interpolation doesn't work, how about we use `bash -c psql .... ` ?